app/controllers/api/v1/organizations_controller.rb

```diff
@@ -3,6 +3,14 @@
 module Api
   module V1
     class OrganizationsController < ApplicationController
+      before_action :authorization
+
+      # GET /organization_id/public
+
+      def public
+        @organization = Organization.find(params[:id])
+        render json: OrganizationSerializer.new(@organization).serializable_hash.to_json
```

Comment:
There is a possibility to specify the fields that you want to show in the JSON output.